app/src/main/java/org/simple/clinic/summary/PatientSummaryConfig.kt

```diff
@@ -17,12 +19,14 @@ data class PatientSummaryConfig(
       val bloodSugarEditableDurationConfig = configReader.long("blood_sugar_editable_duration_in_seconds", 3600)
       val isDiabetesEnabled = configReader.boolean("diabetes_enabled", false)
       val numberOfMeasurementsToDisplay = configReader.long("number_of_measurements_to_display_in_summary", 3)
+      val isBloodSugarEditable = BuildConfig.DEBUG
```

Comment:
Since this is a feature flag and we're turning an entire feature `ON`/`OFF` based on it, we should name this to expose this intent. What do you think of `bloodSugarEditFeatureEnabled`?